packages/ag-charts-community/src/chart/interaction/interactionManager.ts

```diff
@@ -9,22 +8,22 @@ import type { DOMManager } from '../dom/domManager';
 import { type PreventableEvent, type Unpreventable, dispatchTypedEvent } from './preventableEvent';
 
 export const POINTER_INTERACTION_TYPES = [
-    'click',
-    'dblclick',
-    'contextmenu',
-    'hover',
-    'drag-start',
-    'drag',
-    'drag-end',
-    'leave',
-    'enter',
-    'page-left',
-    'wheel',
-] as const;
-
-const FOCUS_INTERACTION_TYPES = ['blur', 'focus'] as const;
-
-const KEY_INTERACTION_TYPES = ['keydown', 'keyup'] as const;
+    'click' as const,
```

Comment:
A single `as const` on the array should be sufficient to keep type safety? (Here and below)

Reply:
`as const` on the array gives a tuple, and `as const` on each element gives an array with the union of all the types